tests/visco_plastic_adiabatic_pressure_in_viscous_creep.prm

```diff
@@ -0,0 +1,149 @@
+# A test for using the adiabatic pressure in creep (diffusion,
+# dislocation, peierls) viscosity calculation for the visco_plastic
+# material model. For example, the adiabatic pressure is 3.3e9 Pa
+# (= rho * g * depth = 3300 * 10 * 100e3) at the bottom. So, the
+# exact diffusion viscosisty ‘eta_diff’ can be calculated by the
+# following equation:
+#
+#   eta_diff = 0.5 * A^(-1/n) * d^(m/n) * exp((E + P_ad*V)/(nRT))
+#            = 7.47e18 Pa*s
+#
+#  A    = 2.37e-15   # diffusion creep prefactor m^3/Pa*s
+#  d    = 1.e-3      # grain size [m]
+#  n    = 1          # stress exponent for diffusion
+#  m    = 3.         # grain size exponents for diffusion
+#  R    = 8.314      # gas constant [J/mol*K]
+#  T    = 1573       # temperature [K]
+#  E    = 375.e3     # activation energy for diffusion [J/mol]
+#  V    = 10.e-6     # activation volume for diffusion [m^3/mol]
+#  P_ad = 3.3e9      # adiabatic pressure at the bottom (Pa)
+
+# Similarly, the exact dislocation viscosity 'eta_disl' and peierls
+# viscosity 'eta_peir' can be obtained by the corresponding equations.
+# The following 'subsection Visco Plastic' provides the relevant parameter
+# values. The material model should return the same viscosity value as
+# the minimum value between these three viscosity values at the bottom.
```

Comment:
This is not exactly true as the output viscosity is the harmonic average of the three values. However, if the values differ significantly (orders of magnitude), then the harmonic average will effectively be equal to the lowest value. I would still reword as suggested below.

Also, in the comments above can you provide the predicted dislocation, Peierls, and composite (average) viscosity? If the results change for some reason, it will be helpful to have these values available for debugging!

Reply:
Addressed these comments